srcpkgs/void-docs/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'void-docs'
+pkgname=void-docs
+version=2020.04.18
```

Comment:
this should probably be something like `0.0.0.20200418` so that if we do tag releases in the future the version here does not create problems.

Reply:
Fair enough